js/script.js

```diff
@@ -0,0 +1,21 @@
+(function ($) {
+
+  /*
+   *  My script code comment
+   */
+  Drupal.behaviors.stanford_soe_r25 = {
+    attach: function (context) {
+      // Show the date select fields after a user chooses a date on old reservation form
+      $('#edit-submitted-event-date-month, #edit-submitted-event-date-finish-month').on('change', function (e) {
```

Comment:
selector is missing `context`
eg: `$('#edit-submitted-event-date-month, #edit-submitted-event-date-finish-month', context)`

Reply:
All of the selectors in an `attach` function should use the `context` scope.